docs/data/material/components/drawers/ResponsiveDrawer.tsx

```diff
@@ -29,9 +29,21 @@ interface Props {
 export default function ResponsiveDrawer(props: Props) {
   const { window } = props;
   const [mobileOpen, setMobileOpen] = React.useState(false);
+  const [isClosing, setIsClosing] = React.useState(false);
+
+  const handleDrawerClose = () => {
+    setIsClosing(true);
+    setMobileOpen(false);
+  };
+
+  const onDrawerTransitionEnd = () => {
```

Comment:
For naming consistency:
```suggestion
  const handleOnDrawerTransitionEnd = () => {
```

Reply:
Actually, per convention it should be:
```suggestion
  const handleDrawerTransitionEnd = () => {
```